edge-lambdas/src/viewer-request/viewer-request.ts

```diff
@@ -34,6 +34,15 @@ export function getHandler(config: Config, s3: S3) {
             request.uri = '/404'
         }
 
+        try {
+            let headers = request.headers
+            const translationCursor = await getTranslationCursor(s3, config)
+            headers = setHeader(headers, TRANSLATION_CURSOR_HEADER, translationCursor)
+            request.headers = headers
```

Comment:
This would also work, right?
```suggestion
            const translationCursor = await getTranslationCursor(s3, config)
            request.headers = setHeader(request.headers, TRANSLATION_CURSOR_HEADER, translationCursor)
```